### PR TITLE
Update spaceapi crate to 0.8 to support v14

### PR DESCRIPTION
**Pull request by rnestler**
## Checklist

- [x] Tests added if applicable
- [x] README updated if applicable
- [x] CHANGELOG updated if applicable
- [x] If a commit includes a breaking change, include the string `[breaking-change]` somewhere in the commit message

### CODE REVIEW

**Comment by rnestler** [comment]
@dbrgn or @g5pw care to take a look at this? 🙂

**Comment by rnestler** [comment]
@dbrgn ping :)

File: examples/with_custom_redis_pool.rs [hunk 1]
```diff
@@ -27,20 +27,23 @@ impl OpenStatusFromRedisModifier {
...
-            Ok(v) => Some(v == "open"),
-            Err(_) => None,
-        };
-        status.state.lastchange = conn.get("state_lastchange").ok();
-        status.state.trigger_person = conn.get("state_triggerperson").ok();
+        let redis_state: RedisResult<String> = conn.get("state_open");
+        if let Some(state) = &mut status.state {
+            state.open = match redis_state {
+                Ok(v) => Some(v == "open"),
+                // v0.13 doesn't allow to remove the state.open field, so we just default to false
```
Comment (g5pw) [inline examples/with_custom_redis_pool.rs:34]: is the 'v0.13' a typo? Anyway, I think removing 'None' as the status is a bit lossy, what should the spaces that don't monitor status put? Should they stay in v13?

Comment (rnestler) [reply examples/with_custom_redis_pool.rs:34]: We want to be compatible with v0.13 and v14 by default. But we fixed this issue in https://github.com/SpaceApi/schema/pull/91 and I'll update spaceapi-rs and spaceapi-server-rs accordingly to keep the old behavior.

Comment (rnestler) [reply examples/with_custom_redis_pool.rs]: See https://github.com/spaceapi-community/spaceapi-rs/pull/94 for the 0.8.1 release of spaceapi-rs.

Comment (rnestler) [reply examples/with_custom_redis_pool.rs]: Done.

**Comment by rnestler** [comment]
@dbrgn @g5pw Ping :slightly_smiling_face:

**Comment by rnestler** [comment]
@dbrgn @g5pw Ping :slightly_smiling_face:

**Pull request merged**